static/js/settings_account.js

```diff
@@ -412,6 +426,7 @@ exports.set_up = function () {
                 overlays.close_modal('change_email_modal');
             },
             error_msg_element: change_email_error,
+            success_msg: i18n.t('Email sent.'),
```

Comment:
@rishig any other suggestion for alert message?

Reply:
good question. Maybe "Check your email (new_email@domain.com) to confirm the new address."